tock/tock/settings/base.py

```diff
@@ -104,3 +105,5 @@
         'rest_framework.authentication.TokenAuthentication',
     ),
 }
+
+VERSION = env.get_credential('CIRCLE_TAG', 'master')
```

Comment:
Defaults to `master` since staging auto-deploys anything on that branch.

Reply:
Why not have align on either `CIRCLE_TAG` or `VERSION`? Why the difference?

Reply:
The difference comes from the fact that `$CIRCLE_TAG` comes from the CI. I was thinking of solving this issue in many ways (e.g. using Bash), but I found that it made more sense to just rely on the variables given to me in the environment for the CI we've all standardized on. If the variable doesn't exist, then it just defaults to the `master` branch which is fine if a Git tag isn't being deployed.

In essence, I wanted to avoid a line like this somewhere in the CI configuration. 🤷‍♂️ 

```yaml
    command: |
        export VERSION=${CIRCLE_TAG:master}
```

Reply:
I was thinking more about having CIRCLE_TAG and VERSION being the same thing... two names for the same thing, but I see why it is this way now.